src/test/fileutils.hpp

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <catch2/catch.hpp>
+
+// Some helper functions for dealing with test data files.
+namespace test {
+
+std::string read_whole_file(const std::string &name) {
```

Comment:
(non-action)
It's frankly embarrassing that in the year of our lord 2021 we have to write this shite, but 100% we do.